cli/src/main/scala/org/apache/celeborn/cli/master/MasterSubcommandImpl.scala

```diff
@@ -0,0 +1,187 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.cli.master
+
+import java.util
+
+import scala.collection.JavaConverters._
+
+import picocli.CommandLine.{Command, ParameterException}
+
+import org.apache.celeborn.cli.config.{CliConfig, CliConfigManager}
+import org.apache.celeborn.rest.v1.model._
+import org.apache.celeborn.rest.v1.model.SendWorkerEventRequest.EventTypeEnum
+
+@Command(name = "master", mixinStandardHelpOptions = true)
+class MasterSubcommandImpl extends Runnable with MasterSubcommand {
+  override def run(): Unit = {
+    if (masterOptions.showMastersInfo) log(runShowMastersInfo)
+    if (masterOptions.showClusterApps) log(runShowClusterApps)
+    if (masterOptions.showClusterShuffles) log(runShowClusterShuffles)
+    if (masterOptions.showTopDiskUsedApps) log(runShowTopDiskUsedApps)
+    if (masterOptions.excludeWorkers) log(runExcludeWorkers)
+    if (masterOptions.removeExcludedWorkers) log(runRemoveExcludedWorkers)
+    if (masterOptions.sendWorkerEvent != null && masterOptions.sendWorkerEvent.nonEmpty)
+      log(runSendWorkerEvent)
+    if (masterOptions.showWorkerEventInfo) log(runShowWorkerEventInfo)
+    if (masterOptions.showLostWorkers) log(runShowLostWorkers)
+    if (masterOptions.showExcludedWorkers) log(runShowExcludedWorkers)
+    if (masterOptions.showShutdownWorkers) log(runShowShutdownWorkers)
```

Comment:
seems it will call the `workerApi.getWorkers` multiple times if both the show*Workes option is true.

Reply:
Is it possible to sort the workers by hostName?

Reply:
@turboFei  both options can never be true. it will always be only one of them. So for example, it can only be 
"master --show-lost-workers" OR "master --show-excluded-workers" and same for the others.

Reply:
yes, we can sort by hostname @turboFei. I will update that.